src/components/Choropleth.vue

```diff
@@ -75,7 +84,10 @@ const layerOnEvents = (feature: Feature) => {
             const country = feature.properties[featureProperties.country];
             // select feature's country, or unselect if click on it when already selected
             const countryToSelect = country === selectedCountryId.value ? "" : country;
-            debounce(() => router.push(`/${APP_BASE_ROUTE}/${selectedIndicator.value}/${countryToSelect}`))();
+            const path = countryToSelect
+                ? `/${APP_BASE_ROUTE}/${selectedIndicator.value}/${countryToSelect}/admin2`
```

Comment:
This should be OK to keep as country route without admin level defined in the path - it should default to admin2

Reply:
yep done